docs/spec/transactions/transaction.json

```diff
@@ -43,6 +43,18 @@
             "type": "string",
             "description": "Keyword of specific relevance in the service's domain (eg: 'request', 'backgroundjob', etc)",
             "maxLength": 1024
+        },
+        "marks": {
+            "type": ["object", "null"],
+            "description": "User defined objects for grouping marks.",
```

Comment:
@jahtalab same here - maybe some more descriptive information comes to your mind?